client/client_pool/src/concord_client_pool.cpp

```diff
@@ -148,7 +156,12 @@ SubmitResult ConcordClientPool::SendRequest(std::vector<uint8_t> &&request,
     ClientPoolMetrics_.rejected_counter++;
     is_overloaded_ = true;
     LOG_WARN(logger_, "Cannot allocate client for" << KVLOG(correlation_id));
-    if (callback) callback(bftEngine::SendResult{SubmitResult::Overloaded});
+    if (callback) {
+      if (serving_candidates == 0 && !clients_.empty())
+        callback(bftEngine::SendResult{SubmitResult::ClientUnavailable});
+      else
+        callback(bftEngine::SendResult{SubmitResult::Overloaded});
```

Comment:
From the Google style guide: "[...] if an if statement has no else or else if clauses, then the curly braces for the controlled statement [...] may be omitted [...]"

Please add curly braces here and in all other places in this PR.

Reply:
Done